webpack/components/SnapshotManagementCard/SnapshotManagementCard.js

```diff
@@ -18,26 +18,28 @@ const SnapshotManagementCard = ({ hostDetails, ...props }) => {
       hostDetails?.capabilities?.includes('snapshot_include_quiesce') || false,
   };
 
-  if (hostDetails?.id && hostDetails?.permissions)
-    children.push(
-      <SnapshotManagement
-        key="SnapshotManagement"
-        host={{ id: hostDetails.id, name: hostDetails.name }}
-        canCreate={hostDetails.permissions.create_snapshots}
-        canUpdate={hostDetails.permissions.edit_snapshots}
-        canRevert={hostDetails.permissions.revert_snapshots}
-        canDelete={hostDetails.permissions.destroy_snapshots}
-        capabilities={capabilities}
-      />
+  if (hostDetails?.uuid) {
```

Comment:
Why is this in another block and not in the line with 
`if (hostDetails?.id && hostDetails?.permissions)`?

Reply:
This is because in case no UUID is set for the host, the Snapshots card should not be displayed at all. Currently in case the condition `(hostDetails?.id && hostDetails?.permissions)` evaluates to `false` the Snapshots card is displayed but stays empty.

And while writing that, i just recognized that it should be the other way round and the condition `(hostDetails?.id && hostDetails?.permissions)` should also be moved one level up to the same level of `(hostDetails?.uuid)`.

What do you think?

Reply:
Changed as discussed